datacats/scripts/compile_less.sh

```diff
@@ -1,2 +1,9 @@
 #!/bin/bash
-lessc $1 > $2
+
+
+lessc /project/target/ckan/ckan/public/base/less/main.less \
+> /project/target/ckan/ckan/public/base/css/main.debug.css
```

Comment:
Yes, but this is mainly a debug feature right? We should do less compiling for prod on the server side, no?